src/system/hot.c

```diff
@@ -71,6 +76,7 @@ void invoke_install_hot_table() {
   // printf("%s %p %p %x %x\n", __FUNCTION__, (void*)install_hot_table, (void*)HOT_TABLE, MAGIC_ADDR[0], MAGIC_ADDR[1]);
   if(vexSystemLinkAddrGet() == (uint32_t)0x03800000 && MAGIC_ADDR[0] == MAGIC0 && MAGIC_ADDR[1] == MAGIC1) {
     install_hot_table(HOT_TABLE);
+
```

Comment:
@HotelCalifornia When you clean up the PR to remove some of the other test changes, can you rm this accidentally added line?